db/migrate/20210726060426_add_fields_to_host_initiators.rb

```diff
@@ -0,0 +1,6 @@
+class AddFieldsToHostInitiators < ActiveRecord::Migration[6.0]
+  def change
+    add_column :host_initiators, :status, :string
```

Comment:
What kind of values will be stored here?  I'm curious if we will need normalization or just want to store the raw values (or both?)

Reply:
included by mistake.  not related to this PR